cves/CVE-2011-3944.yml

```diff
@@ -236,8 +236,8 @@ subsystem:
     If this involves fixing multiple subsystems, you can make this an array.
 
     In 'answer', explain how you arrived at this determination.
-  answer:
-  name:
+  answer: It was located within the libavcodec folder with no sub-folders.
```

Comment:
Smacker is also a codec, so that's another reason why it was placed in the libavcodec folder

Reply:
Updated